source/val/validate_memory.cpp

```diff
@@ -547,6 +547,7 @@ spv_result_t ValidateVariable(ValidationState_t& _, const Instruction* inst) {
            << "PhysicalStorageBufferEXT must not be used with OpVariable.";
```

Comment:
Does _EXT_ mean decoration?
What is _PhysicalStorageBufferEXT_? is it the same as StorageBuffer, Shared externally, readable and writable, visible across all functions in all invocations in all work groups. Graphics storage buffers (buffer blocks)?
I think to fix this bug I need to simply remove the three nested if-conditions that is being applied to the code in lines 559 - 572, update the error message, tests and ... . Does it sound reasonable?

Reply:
EXT is the suffix generally used with tokens associated with SPV_EXT_ extensions. In this case, the extension is SPV_EXT_physical_storage_buffer. PhysicalStorageBufferEXT is a storage buffer that has a physical64 addressing model.